test/scripts/stylelint/stylelint.spec.ts

```diff
@@ -1,53 +1,41 @@
-import spawn from "cross-spawn";
 import path from "path";
+import { runExosScript } from "../../test-utils";
 import type { SpawnSyncReturns } from "child_process";
 
+const MOCKS_FOLDER_PATH = path.resolve(__dirname, "./mocks");
+const errorMessage: string = "❌ There were errors while running stylelint.";
+
 describe("stylelint E2E", () => {
-  let scriptPath: string;
-  let fileToTestPath: string;
   let args: string[];
   let results: SpawnSyncReturns<string>;
 
-  beforeAll(() => {
-    scriptPath = path.resolve(process.cwd(), "./lib/index.js");
-  });
-
-  describe("when linting a SASS file with CSS Modules with no errors", () => {
+  describe("when stylelinting a file with no errors", () => {
     beforeAll(() => {
-      fileToTestPath = path.resolve(process.cwd(), "./test/scripts/stylelint/mocks/sass-and-css-modules.scss");
-      args = ["stylelint", `--files=${fileToTestPath}`];
-      results = runScript(scriptPath, args);
+      args = ["stylelint", `--files=${MOCKS_FOLDER_PATH}/sass-and-css-modules.scss`];
+      results = runExosScript(args);
     });
 
-    it("should not output stylelint errors", () => {
-      const hasErrorMessage = results.stdout.indexOf("❌ There were errors while running stylelint.") !== -1;
-      expect(hasErrorMessage).toBe(false);
+    it("should not output errors", () => {
+      expect(results.stdout.includes(errorMessage)).toBe(false);
     });
 
     it("should exit with status code 0", () => {
       expect(results.status).toBe(0);
     });
   });
 
-  describe("when linting a SASS file with CSS Modules with errors", () => {
+  describe("when stylelinting a file with errors", () => {
     beforeAll(() => {
-      fileToTestPath = path.resolve(process.cwd(), "./test/scripts/stylelint/mocks/failing-test-cases.scss");
-      args = ["stylelint", `--files=${fileToTestPath}`];
-      results = runScript(scriptPath, args);
+      args = ["stylelint", `--files=${MOCKS_FOLDER_PATH}/failing-test-cases.scss`];
+      results = runExosScript(args);
     });
 
-    it("should output stylelint errors", () => {
-      const hasErrorMessage = results.stdout.indexOf("❌ There were errors while running stylelint.") !== -1;
-      expect(hasErrorMessage).toBe(true);
+    it("should output errors", () => {
+      expect(results.stdout.includes(errorMessage)).toBe(true);
     });
 
     it("should exit with status code 0", () => {
```

Comment:
Why do both runs, the positive and the negative ones return 0?

Reply:
Fixed